angular-rails-app/lib/tasks/db_connector.rake

```diff
@@ -322,25 +324,25 @@ namespace :db_connector do
     add_indexes(indexes)
 
     v_types = [
-      "IMD-IMMEDIATE EMERGENCY",
-      "COMPBLD-STRUCTURALLY COMPROMISED BUILDING",
-      "HBLVIO-HIGH PRESSURE BOILER",
-      "LL1080-LOCAL LAW 10/80 - FACADE",
-      "P-PLUMBING",
-      "EGNCY-EMERGENCY",
-      "UB-UNSAFE BUILDINGS",
-      "IMEGNCY-IMMEDIATE EMERGENCY",
-      "LBLVIO-LOW PRESSURE BOILER",
-      "LL1081-LOCAL LAW 10/81 - ELEVATOR",
-      "LL6291-LOCAL LAW 62/91 - BOILERS",
-      "C-CONSTRUCTION",
-      "B-BOILER",
+      "IMD", # -IMMEDIATE EMERGENCY",
+      "COMPBLD", # -STRUCTURALLY COMPROMISED BUILDING",
+      "HBLVIO", # -HIGH PRESSURE BOILER",
+      "LL1080", # -LOCAL LAW 10/80 - FACADE",
+      "P", # -PLUMBING",
+      "EGNCY", # -EMERGENCY",
+      "UB", # -UNSAFE BUILDINGS",
+      "IMEGNCY", # -IMMEDIATE EMERGENCY",
+      "LBLVIO", # -LOW PRESSURE BOILER",
+      "LL1081", # -LOCAL LAW 10/81 - ELEVATOR",
+      "LL6291", # -LOCAL LAW 62/91 - BOILERS",
+      "C", # -CONSTRUCTION",
+      "B", # -BOILER",
     ]
 
     in_list =  "\"" + v_types.join("\", \"") + "\""
 
     puts "Pulling dob violations..."
-    sql = "SELECT * FROM dob_violations WHERE violation_type IN (#{in_list})"
+    sql = "SELECT * FROM dob_violations WHERE violation_type_code IN (#{in_list})"
```

Comment:
Nice, this'll probably perform better. You should change the `add_indexes` params above to match this column.